conversion/conversion.go

```diff
@@ -444,17 +517,42 @@ func CreateDatabase(project, instance, dbName string, conv *internal.Conv, out *
 	if _, err := op.Wait(ctx); err != nil {
 		return "", fmt.Errorf("createDatabase call failed: %w", analyzeError(err, project, instance))
 	}
-	fmt.Fprintf(out, "done.\n")
+	fmt.Fprintf(out, "Created database successfully.\n")
 	return fmt.Sprintf("projects/%s/instances/%s/databases/%s", project, instance, dbName), nil
 }
 
+// UpdateDatabase updates the DDL of the spanner DB and return the path.
+func UpdateDatabase(project, instance, dbName string, conv *internal.Conv, out *os.File) (string, error) {
+	fmt.Fprintf(out, "Updating schema for database %s in instance %s ... \n", dbName, instance)
+	ctx := context.Background()
+	adminClient, err := database.NewDatabaseAdminClient(ctx)
+	if err != nil {
+		return "", fmt.Errorf("can't create admin client: %w", analyzeError(err, project, instance))
+	}
+	defer adminClient.Close()
+
+	dbURI := fmt.Sprintf("projects/%s/instances/%s/databases/%s", project, instance, dbName)
+	op, err := adminClient.UpdateDatabaseDdl(ctx, &adminpb.UpdateDatabaseDdlRequest{
+		Database:   dbURI,
+		Statements: conv.SpSchema.GetDDL(ddl.Config{Comments: false, ProtectIds: true, Tables: true, ForeignKeys: false}),
```

Comment:
why is ForeignKeys false ? Is this handled elsewhere ?

Reply:
Yes foreign keys we create at the very end after data migration. This only creates tables and indexes.

Reply:
I think it's fair to add comment to the same effect to help future readers understand this.